webpack/assets/javascripts/react_app/components/common/AutoComplete/index.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import Downshift from 'downshift';
+import classnames from 'classnames';
+import { InputGroup } from 'react-bootstrap';
+import { Button } from 'patternfly-react';
+import PropTypes from 'prop-types';
+
+import './AutoComplete.scss';
+
+export const renderItems = ({
+  items, highlightedIndex, selectedItem, getItemProps,
+}) => (
+  <div className="autocomplete-dropdown-container">
```

Comment:
You can build this container with `patternfly-react`, i think you won't need to write any class or css:
[MenuItem](https://rawgit.com/patternfly/patternfly-react/gh-pages/index.html?knob-Disabled=false&knob-Style=default&knob-Show%20Labels=false&knob-Show%20Help=true&knob-Show%20Loading=false&selectedKind=MenuItem&selectedStory=MenuItem&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs)